### ISSUE TITLE
Track commute, build reporting views/queries

### ISSUE THREAD

**Issue by hozn** [freezingsaddles/freezing-web#19]
Shawn says:
I was hoping to award a pointless prize based on the ratio of 'commute' rides to total rides for the duration of the competition.  Is that data available?  I've seen the 'commute' banner on strava rides recently and was hoping that info was included in data that could be pulled down.

So i guess i'd just need two numbers for each rider:

total number of rides
total number of rides flagged with 'commute'

**Comment by jrenaut** [comment]
I'd worry that people aren't that consistent with using that tag. Also, I'll throw off any numbers since I generally stop and save a ride every time I get off my bike at a destination (I have 161 rides this year, twice as many as the person with the 10th most) and work 100% from home. 

Although you're in luck, looks like it's a true/false on [the activity](https://strava.github.io/api/v3/activities/)

**Comment by hozn** [comment]
Yeah, I know the attrib is there.  People can always go back and add it.  I agree it's not going to be 100% accurate, though, since I sometimes forget.  So it'll only be as good as the data.  We could get really fancy and match on other activities that have the same geometry as those matched by commutes ... but we won't actually do that. :)

**Comment by jrenaut** [comment]
Although that would be a fun project if someone was going to pay me for the time I spent on it...

**Comment by jrenaut** [comment]
Just realized you were quoting a feature request, not adding your own thoughts. Makes more sense, I was a little surprised that you'd be unaware of the commute flag.

**Comment by hozn** [comment]
yeah, i forgot to indicate that in the ticket :)

**Comment by hozn** [comment]
Apparently we're already tracking commutes (commute bool) in the db too.  So I guess all that is left is to make a chart or something.

**Comment by merlinorg** [comment]
OBE. We had a #553 in 2026. The `commute` flag isn't, by itself, useful for a competition, because people disagree on what constitutes a commute.
